src/app/legacy/components/OptimoPromos/Image/index.jsx

```diff
@@ -1,22 +1,23 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { string, number } from 'prop-types';
-import ImageWithPlaceholder from '#containers/ImageWithPlaceholder';
-import StyledImage from './index.styles';
+import { RequestContext } from '#contexts/RequestContext';
+import IMAGE from '../../../../components/Image';
```

Comment:
Just because it collides with the component defined in this file that is already called `Image`

Reply:
oh did not notice that. Maybe not in the scope of this, but might be good to rename the `Image` component to `PromoImage`. It might be confusing to have 2 components named Image.